config/7.1.0/securebanking/ig/scripts/groovy/AddDetachedSig.groovy

```diff
@@ -36,7 +36,13 @@ next.handle(context, request).thenOnResult({ response ->
     signingManager.newSigningHandler(purpose).then({ signingHandler ->
         logger.debug(SCRIPT_NAME + "Building of the JWT started")
 
-        JwtClaimsSet jwtClaimsSet = new JwtClaimsSet(response.getEntity().getJson())
+        JwtClaimsSet jwtClaimsSet
+        // We get content empty on submit file payment API
+        if (response.getEntity().isRawContentEmpty()) {
+            jwtClaimsSet = new JwtClaimsSet()
+        } else {
```

Comment:
When the content is empty we need to return an error, doesn't make sense process the request because never will match with the initiation object send in the consent previously.

Reply:
It's not an error case. It'a a success case where the response payload is empty. We need to create am empty signature from what I understand from the spec (on the response of submit file payment file the response on the API is 200 OK with empty body). This is the case that was failing.